File: SchoolManagement.Api/Models/SourceInformation.cs
﻿using System;
using System.Collections.Generic;

namespace SchoolManagement.Api.Models
{
    public partial class SourceInformation
    {
        public int SourceInformationId { get; set; }
        public int? BaseSchoolNameId { get; set; }
        public int? BookInformationId { get; set; }
        public string SourceName { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public DateTime? OrderDate { get; set; }
        public DateTime? ReceivedDate { get; set; }
        public string Remarks { get; set; }
        public int? MenuPosition { get; set; }
        public string CreatedBy { get; set; }
        public DateTime DateCreated { get; set; }
        public string LastModifiedBy { get; set; }
        public DateTime? LastModifiedDate { get; set; }
        public bool IsActive { get; set; }

        public virtual BaseSchoolName BaseSchoolName { get; set; }
        public virtual BookInformation BookInformation { get; set; }
    }
}

File: ClientApp/src/app/book-management/book-management-tab/models/SourceInformation.ts
export interface SourceInformation {
    sourceInformationId: number;
    baseSchoolNameId: number;
    bookInformationId: number;
    sourceName: string;
    address: string;
    phone:number;
    email: string;
    orderDate: Date;
    receivedDate: Date;
    remarks: string;
    //menuPosition: string;
    isActive: boolean;
}